Automattic/wp-calypso
# Fix reader post card comment count to update when comments change.

… may properly use its selector to get the comment count



Related to # pe7F0s-17X-p2

## Proposed Changes
* Passes the `post` object to the CommentButton component. This allows the component to properly use the selector already in its mapStateToProps function that will update comment counts appropriately on state change. We continue to pass `commentCount` as it gives a default value to render before comments state is initialized, and is used as a fallback by the component.

## Testing Instructions



* Run this branch.
* Post an inline comment in the reader and ensure the comment count next to the comment icon in the post card updates.

## Pre-merge Checklist
